src/components/screens/editor/editor-screen-drawer.tsx

```diff
@@ -187,6 +187,10 @@ const EditorScreenDrawer: FC<Props> = ({
             <FormControlLabel value={ true } control={ <Radio/> } label={ strings.editorScreen.drawer.formTemplateSosmeta }/>
             <FormHelperText>
               { strings.editorScreen.drawer.formTemplateSosmetaHelper }
+              <Link href="https://sosmeta.thl.fi/document-definitions/list" target="_blank">
+                { strings.editorScreen.drawer.formTemplateSosmetaLink }
+              </Link>
+              { strings.editorScreen.drawer.formTemplateSosmetaDot }
```

Comment:
I think this can be removed after all, after colon, it's not really needed.

Reply:
done.